src/main/java/com/celements/course/CourseScriptService.java

```diff
@@ -87,8 +89,8 @@ public String getCourseTypeClass() {
   }
 
   DocumentReference getDocRefForFullName(String courseFN) {
```

Comment:
Please mark getDocRefForFullName as deprecated with hint to org.xwiki.model.internal.scripting.ModelScriptService